bundles/org.eclipse.core.resources/src/org/eclipse/core/internal/resources/Workspace.java

```diff
@@ -2253,6 +2257,7 @@ public IStatus open(IProgressMonitor monitor) throws CoreException {
 		//finally register a string pool participant
 		stringPoolJob = new StringPoolJob();
 		stringPoolJob.addStringPoolParticipant(saveManager, getRoot());
+		initializePreferenceLookupOrder();
```

Comment:
Is there a reason to move this call down, especially after notification and refresh manager init?

Reply:
The reason is that I forgot to revert this :-\

Reply:
This should now be fixed.